From 8f0ce37cda2ad5b81e3489a241d4e1453686e661 Mon Sep 17 00:00:00 2001
From: Chen Chong <chongc@qca.qualcomm.com>
Date: Thu, 04 Jul 2013 03:40:40 +0000
Subject: ar3k: add Bluetooth support for QCA98XX/QCA24XX

add patch & syscfg for Qualcomm Atheros QCA98XX/QCA24XX

Signed-off-by: Chen Chong <chongc@qca.qualcomm.com>
Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
diff --git a/WHENCE b/WHENCE
index d7fba9d..3956c09 100644
--- a/WHENCE
+++ b/WHENCE
@@ -1928,6 +1928,8 @@ File: ar3k/ramps_0x11020000_40.dfu
 File: ar3k/AthrBT_0x01020201.dfu
 File: ar3k/ramps_0x01020201_26.dfu
 File: ar3k/ramps_0x01020201_40.dfu
+File: ar3k/AthrBT_0x41020000.dfu
+File: ar3k/ramps_0x41020000_40.dfu
 
 Licence: Redistributable. See LICENCE.atheros_firmware for details
 
diff --git a/ar3k/AthrBT_0x41020000.dfu b/ar3k/AthrBT_0x41020000.dfu
new file mode 100755
index 0000000..b1470d8
--- /dev/null
+++ b/ar3k/AthrBT_0x41020000.dfu
Binary files differ
diff --git a/ar3k/ramps_0x41020000_40.dfu b/ar3k/ramps_0x41020000_40.dfu
new file mode 100755
index 0000000..77383bb
--- /dev/null
+++ b/ar3k/ramps_0x41020000_40.dfu
Binary files differ
--
cgit v0.9.2
